easier68k/core/opcodes/Bcc.py

```diff
@@ -364,4 +364,403 @@ def disassemble_instruction(cls, data: bytes):
+
+    def conditional(self, simulator: M68K):
+        n = simulator.get_condition_status_code(ConditionStatusCode.N)
+
+        if n:
+            return True
+
+        return False
+
+    @classmethod
+    def command_matches(cls, command: str):
+        return opcode_util.command_matches(command, 'BMI')
+
+    @classmethod
+    def disassemble_instruction(cls, data: bytes):
+        return bcc_disassemble_instruction(data, BMI, ord('\x0B'))
+
+    @classmethod
+    def from_str(cls, command: str, parameters: str):
+        return bcc_from_str(command, parameters, BMI)
+
+
+class BGE(branch_code):
+    """
+    The BGE opcode compiles down to 6Cxx xxxx xxxx
+    The conditional here is N AND V OR NOT N AND NOT V
+    """
+
+    cond_code = '\x0C'
+
+    def conditional(self, simulator: M68K):
+        n = simulator.get_condition_status_code(ConditionStatusCode.N)
+        v = simulator.get_condition_status_code(ConditionStatusCode.V)
+
+        if n and v:
+            return True
+
+        if not n and not v:
+            return True
+
+        return False
+
+    @classmethod
+    def command_matches(cls, command: str):
+        return opcode_util.command_matches(command, 'BGE')
+
+    @classmethod
+    def disassemble_instruction(cls, data: bytes):
+        return bcc_disassemble_instruction(data, BGE, ord('\x0C'))
+
+    @classmethod
+    def from_str(cls, command: str, parameters: str):
+        return bcc_from_str(command, parameters, BGE)
+
+
+class BLT(branch_code):
+    """
+    The BLT opcode compiles down to 6Dxx xxxx xxxx
+    The conditional here is N AND NOT V OR NOT N AND V
+    """
+
+    cond_code = '\x0D'
+
+    def conditional(self, simulator: M68K):
+        n = simulator.get_condition_status_code(ConditionStatusCode.N)
+        v = simulator.get_condition_status_code(ConditionStatusCode.V)
+
+        if n and v:
+            return True
+
+        if not n and not v:
+            return True
+
+        return False
+
+    @classmethod
+    def command_matches(cls, command: str):
+        return opcode_util.command_matches(command, 'BLT')
+
+    @classmethod
+    def disassemble_instruction(cls, data: bytes):
+        return bcc_disassemble_instruction(data, BLT, ord('\x0D'))
+
+    @classmethod
+    def from_str(cls, command: str, parameters: str):
+        return bcc_from_str(command, parameters, BLT)
+
+
+class BGT(branch_code):
+    """
+    The BGT opcode compiles down to 6Exx xxxx xxxx
+    The conditional here is N AND V AND NOT Z OR NOT N AND NOV AND NOT Z
+    """
+
+    cond_code = '\x0E'
+
+    def conditional(self, simulator: M68K):
+        n = simulator.get_condition_status_code(ConditionStatusCode.N)
+        v = simulator.get_condition_status_code(ConditionStatusCode.V)
+        z = simulator.get_condition_status_code(ConditionStatusCode.Z)
+
+        if n and v and not z:
+            return True
+
+        if not n and not v and not z:
+            return True
+
+        return False
```

Comment:
Could be simplified to `return (n == v) and not z`

Reply:
not so sure that is true.
The manual does not say that  N & V & ~Z are valid.

Reply:
but isn't that exactly the condition you are checking on line 709?